org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "org.arrl.trustedqsl",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
```

Comment:
`share=ipc` is  needed for X11.

Reply:
Added - I missed this in the last round.